ChangeLog.txt

```diff
@@ -1,8 +1,31 @@
 Next v0.1.X
 -----------
 
+Cultures:
+
+- Added namelist for Vandal culture
+- Added dynasty names for Roman, Romano-Gallic, Romano-Hispanic, Romano-Thracian, Romano-Illyrian and Romano-Aramean cultures
+- Added many new name equivalencies and fixed some existing ones
+
+Localization:
+- Added numerous cultural title localization for Britannia, Gallia, Italia, Hispania, Africa, Eastern Roman Empire, Persia, Pontic-Caspian steppe, Tamazgha and Germania Magna
+- Fixed some typos among the title names
+
+History:
+- Expanded Nepos' Dalmatia based on a source, which states that it included lands between Drina and Sava. In addition, Dalmatia is now controls Epidaurum instead of ERE
+- Some work on character cultures:
+   - Nepos and Glycerius were made Romano-Illyrian because they were natives to Dalmatia
+   - All the popes are now have appropriate cultures to reflect their origins
+   - Macrinus, Aemilian and some Roman families were made Romano-African
+- Made Burdigala Romano-Gallic
```

Comment:
Isn't this already covered by @LT-Rascek's pull request?

Reply:
> Isn't this already covered by @LT-Rascek's pull request?

I believe that we simultaneously added that change to our repositories after I suggested it on Discord.